Framework/PythonInterface/mantid/geometry/src/Exports/Goniometer.cpp

```diff
@@ -28,6 +28,7 @@ GNU_DIAG_OFF("unused-local-typedef")
 // Seen with GCC 7.1.1 and Boost 1.63.0
 GNU_DIAG_OFF("conversion")
 // define overloaded functions
+// cppcheck-suppress unknownMacro
```

Comment:
Seems to be an issue with cppcheck not being able to find boost macros, will create an issue for it.

Reply:
Yes, that situation occurred for us as well.